Week01/Problem03/ylu_03.py

```diff
@@ -0,0 +1,13 @@
+n = 600851475143
+answer = 2
+
+while n > answer:
+    if n % answer == 0:
+        while n % answer == 0:
+            n = n / answer
+    #divide all of the same factor, so the next factor found by this loop will still be a prime number
+    #python doesn't need to type 'else' & 'end'
```

Comment:
That's correct...Python determines codeblocks by indentation, rather than by characters (C, C++, etc.) or reserved words (Bash, Perl, etc.).